docs/_components/alerts.md

```diff
@@ -63,10 +63,10 @@ Visit the [USWDS Alerts component](https://designsystem.digital.gov/components/a
 {% endcapture %}
 {% include helpers/code-example.html code=example %}
 
-### Other
+### Default
```

Comment:
@aduth Are we calling the gray version as "Default" from now on, correct?

Reply:
I think it makes sense to, since that is how these styles are applied.

Basically the USWDS CSS looks like:

```css
.usa-alert {
  // base
}

.usa-alert--info {
  // primary
}

// etc.
```

So this variant is the one without any BEM class modifiers (`--info`, etc), i.e. default.